internal/manifests/pod/compare_test.go

```diff
@@ -90,7 +91,7 @@ var _ = Describe("podtemplate", func() {
 		})
 
 		It("should recognize podtemplates as the same", func() {
-			Expect(ArePodTemplateSpecDifferent(lhs, rhs)).To(BeFalse())
+			Expect(ArePodTemplateSpecEqual(lhs, rhs)).To(BeFalse())
```

Comment:
shouldn't all of these be flipped now?

Reply:
The whole test file was not included in our suite. After including this it was full or errors thanks for ginkgo and global var misuse. I rewrote this in `internal/manifest/pod/compare_test.go` without ginkgo.